api/views.py

```diff
@@ -92,3 +92,14 @@ def insert_internship():
     Returns the login page: which renders insert_listing.html (insert listing page)
     """
     return render_template('insert_listing.html', page_title='Request Internship Listing')
+
+
+# Renders reset password form:
+@views.route('/login/reset-password', methods=['GET'])
+def reset_password():
+    """Reset Password page for users. (Pending Auth)
+    This function runs whenever the reset-password page ('/reset-password') is requested.
+        ex) -> localhost:5000/login/reset-password
+    Returns the reset password page: which renders reset.html (reset form)
+    """
+    return render_template('reset.html', page_title='Reset Password')
```

Comment:
Not an error but calling the template "reset_password.html" would be more intuitive